templates/zerver/about.html

```diff
@@ -0,0 +1,124 @@
+{% extends "zerver/portico.html" %}
+{% block portico_content %}
+
+<h1>About Zulip</h1>
+
+<p>
+  This server is an installation of
+  the <a href="https://zulip.org">Zulip open source group chat
+  software</a>.  Written in Python and using the Django web framework,
+  Zulip has an extensive real-time messaging featureset, including
+  both group and private messaging, conversation streams, powerful
+  search, drag-and-drop file uploads, image previews, audible
+  notifications, missed-message emails, markdown formatting, desktop
+  and mobile apps, dozens of integrations, and much, much more.
+</p>
+
+<p>
+  Zulip was designed from the ground up to optimize the productivity
+  of discussions and real-time decision-making.  A key feature that
+  drives this is Zulip's unique threading model and system for
```

Comment:
maybe say "topic threading model" here?  (When I first read it, I thought you mean system threads.)